src/docs/product/issues/index.mdx

```diff
@@ -32,6 +32,26 @@ From the **Issues** page, you can begin to triage. The page is organized into ta
 
 Learn more about triaging issues and their different states in [Issue States and Triage](/product/issues/states-triage/).
 
+### Display Options
+
+You can use the display dropdown in the issues page to see error counts as a % of [sessions](/product/releases/health/#session).
+
+![Issues display dropdown](issues-events-as-percentage-dropdown.png)
+
+For this option to be enabled you need to have selected a project that has [release health set up](/product/releases/health/setup) and need to have issues and sessions present in the selected period.
+
+With `Events as %` option selected, instead of event counts you will see the issues' events as a % of overall sessions for the project.
+
+<Note>
+
+  The "Events as %" display option is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.
+
+  If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
+
+</Note>
+
+![Issues as a percentage of sessions](issues-events-as-percentage-counts.png)
```

Comment:
Check with @robinrendle to see if we can get a better screenshot here. This one is so large that it's hard to get a sense of what the focus should be. He might have a better idea of how to frame this.